packages/neuron-wallet/src/services/sync/renderer-params.ts

```diff
@@ -0,0 +1,5 @@
+import { remote } from 'electron'
+
+const { app }: { app: any } = remote
+const { syncBlockTask } = app
```

Comment:
Instead of defining `syncBlockTask` on `app` object, we should try using `remote.require('sync-block-task')` and see if that works well.

Reply:
export there params in `sync-block-task/params.ts` and require them by `remote.require`